web-client/src/modules/request/index.ts

```diff
@@ -1,10 +1,92 @@
+import {
+  ContainerOutlined,
+  FileAddOutlined,
+  FileDoneOutlined,
+  FileProtectOutlined,
+  FileSearchOutlined,
+  FileSyncOutlined,
+} from '@ant-design/icons';
+import { uniqueId } from 'lodash';
+import React from 'react';
+import Location from 'react-app-location';
 import { Module } from 'src/types/module';
 
+import { MenuItem } from '../../types/menu-item';
 import ContentPage from './pages/ContentPage';
+import { NewRequestLocation } from './pages/routes/NewRequestRoute/constants';
+
+// TODO (remove link id): ask help to find a better way to keep track of ids
+export const menuItems: Array<MenuItem> = [
```

Comment:
@ashwinkjoseph This is the only resource for recursive menu generation. But, I don't think that it's the right place. Any advice about where to store this static data?

Reply:
I do think that prop drilling in these cases make the data flow clearer. I like using Context for more global things such as themes